pkg/server/server_test.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2018 The OpenEBS Author
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server_test
+
+import (
+	"net"
+	"testing"
+
+	"github.com/openebs/node-disk-manager/pkg/server"
+)
+
+func TestStartHttpServer(t *testing.T) {
+
+	ErrorMessages := make(chan error)
+	go func() {
```

Comment:
@ShovanMaity These changes LGTM , it will be good if you add comments here.

Reply:
@utkarshmani1997 I will add.

Reply:
added